### DISCUSSION TITLE
Bug Bash

### DISCUSSION THREAD

**Discussion by SydneyhSmith** [PowerShell/PSResourceGet#1289] [General]
We will be having a LIVE PUBLIC BUG BASH July 27th 9am-10am pdt (Seattle time) for PSResourceGet...more info and a Teams join link to follow🐛 🥳 (there will be async instructions provided afterwards for those who aren't able to join live)

**Comment by SydneyhSmith** [comment]
To join the event on Thursday use the link 
https://aka.ms/PSBugBash

**Reply by SimonWahlin** [reply]
Thanks for a fun event! I hope you'll do this again.
